### PR TITLE
Cleanup go vet warnings

### PR DESCRIPTION
**Pull request by gliptak**
https://goreportcard.com/report/github.com/constabulary/gb

### CODE REVIEW

File: cmd/gb/gb_test.go [hunk 1]
```diff
@@ -451,7 +451,7 @@ func (t *T) mustBeEmpty(root string) {
 		found = append(found, path)
 		return nil
 	}
-	err = filepath.Walk(root, fn)
+	_ = filepath.Walk(root, fn)
```
Comment (davecheney) [inline cmd/gb/gb_test.go]: this is a bug, err is not being checked.

Comment (gliptak) [inline cmd/gb/gb_test.go]: https://github.com/constabulary/gb/blob/master/cmd/gb/gb_test.go#L449 would have failed the test already

Comment (davecheney) [inline cmd/gb/gb_test.go]: what about 

filepath.Walk(root, fn)

without assigning the unused value to anything?

Comment (gliptak) [inline cmd/gb/gb_test.go]: changed

**Pull request merged**